test/language/expressions/generators/length-property-descriptor.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Generator objects should define a `length` property.
+includes: [propertyHelper.js]
+es6id: 25.2.4
+---*/
+
+var g = function*() {};
+
+verifyNotEnumerable(g, 'length');
+verifyNotWritable(g, 'length');
+verifyConfigurable(g, 'length');
```

Comment:
Unlike `name`, there aren't any tests for the value of the length property. I agree it's worth adding a test now, but I'll wait on Brian's response to https://github.com/tc39/test262/pull/237#discussion_r29007374 before acting on this (since I want to include the new assertions in a way that is consistent with the others).